server/backend/db_backend.go

```diff
@@ -279,10 +279,6 @@ func (mb *MongoBackend) importTx(ctx context.Context, tx *types.Transaction, blo
 		}
 	}
 
```

Comment:
The Upsert below was redundant now that ensureReceipt is always called prior, which does it's own write.